packages/serverpod/lib/src/server/server.dart

```diff
@@ -65,6 +65,9 @@ class Server {
   /// Central message dispatch for real time messages.
   MessageCentral messageCentral = MessageCentral();
 
+  /// Define CORS headers to allow cross-origin requests.
+  final Map<String, dynamic> headers;
```

Comment:
This defines any type of headers, just not for CORS, so the documentation should reflect this. Perhaps a more descriptive name would be good too. `httpResponseHeaders`?

`/// Http headers used by all API responses. Defaults to allowing any cross origin resource sharing (CORS).`